src/components/text-input-with-copy-button/TextInputWithCopyButton.scss

```diff
@@ -27,11 +27,19 @@
         transition: background-color .5s ease, border-color .5s ease;
     }
 
+    .bdl-Button:not(:last-child),
+    .btn:not(:last-child) {
+        margin: 0;
+        border-left: 0;
+        border-radius: 0;
+        transition: background-color .5s ease, border-color .5s ease;
+    }
+
     &.copy-success {
-        .btn,
-        .btn:hover,
-        .btn:active,
-        .btn:active:hover {
+        .btn:last-child,
+        .btn:last-child:hover,
+        .btn:last-child:active,
+        .btn:active:last-child:hover {
```

Comment:
I think we can remove this selector `.btn:active:hover`. it should be the same as the two above it